soc/riscv/riscv-privilege/gd32vf103/Kconfig.defconfig.gd32vf103

```diff
@@ -34,9 +34,6 @@ config NUM_IRQS
 	default 87 if  NUCLEI_ECLIC
 	default 16 if !NUCLEI_ECLIC
 
-config FLASH_BASE_ADDRESS
```

Comment:
Since you removed this, The CONFIG_FLASH_BASE_ADDRESS will be detected correctly.